router/auth.go

```diff
@@ -84,6 +84,42 @@ func authPassThrough(log *log.Logger, w http.ResponseWriter, req *http.Request,
 
 func authInjectECS(log *log.Logger, w http.ResponseWriter, req *http.Request, target *targetPassThroughAuth) bool {
 	req.SetBasicAuth(target.config.Username, target.config.Password)
+	var didWhat string
+	var toWhat string
+	actionUrl := strings.Split(req.URL.String(), "/") // Example: "ecs/ACCESS/FWVERSION/" or "ecs/sam/ForceSim1/"
+	if len(actionUrl) == 5 {
+		switch actionUrl[2] {
+		case "ACCESS":
+			didWhat = actionUrl[3]
+			toWhat = "site gate: " + actionUrl[4]
+		case "sam":
+			didWhat = actionUrl[3]
+			toWhat = "site: " + actionUrl[4]
+		default:
+			didWhat = req.URL.String()
+			toWhat = req.URL.String()
+		}
+	} else {
+		didWhat = req.URL.String()
+		toWhat = req.URL.String()
+	}
+	b := new(bytes.Buffer)
```

Comment:
Check out [bytes.NewBufferString](https://golang.org/pkg/bytes/#NewBufferString) to replace `b` with a single statement since (if) `b` is not used again in the code?